interfaces/builtin/desktop.go

```diff
@@ -300,6 +318,25 @@ dbus (send, receive)
       peer=(label=unconfined),
 `
 
+var desktopPermanentSlotAppArmor = `
```

Comment:
This seems to be unused, I guess it's a leftover from another branch?

Reply:
I've added an extra revision that sorts this out. I'd been testing these additional AppArmor rules ad-hoc, and noticed I'd missed this when rebuilding a new image.